views/partials/home/banner.mako

```diff
@@ -13,8 +13,10 @@
 % for cur_show_list in show_lists:
     <% cur_list_type = cur_show_list[0] %>
     <% my_show_list = list(cur_show_list[1]) %>
-    % if cur_list_type == "Anime":
-        <h1 class="header">Anime List</h1>
+    % if len(show_lists) > 1:
+        % if len(show_lists) > 1:
```

Comment:
Why the double check here? 😕

Reply:
I'll fix that. Sorry. Didn't notice. Still have it checked out anyway.

Reply:
Np, thanks!